mm2src/mm2_main/tests/docker_tests/swap_watcher_tests.rs

```diff
@@ -255,6 +255,7 @@ fn start_swaps_and_get_balances(
 }
 
 #[test]
+#[ignore]
```

Comment:
Is it necessary to ignore the tests?

Reply:
Yes otherwise they will all fail with disabled watchers

Reply:
you can keep the watchtower configurable from coins configuration only for debug builds, so releases will still have watchtower as not activated but the test will not need to be disabled(so we ensure not breaking anything on watchtowers while making changes on mm2)

Reply:
you can alternatively use `#[cfg(test)] ` attribute to enable watchers in tests only.

Reply:
> #[cfg(test)] does not work for docker tests

Sorry, forgot about that :)

>  used environment variables as a quick fix to enable the tests again

Seems fine for now until the fix